python3/36.py

```diff
@@ -0,0 +1,53 @@
+class Solution:
+    def isValidSudoku(self, board: List[List[str]]) -> bool:
```

Comment:
**PaiR(PR AI Reviewer) :** You should add a space after the comma in `(i//3,j//3)` for readability.